code/datums/wounds/burns.dm

```diff
@@ -261,17 +226,26 @@
 	COOLDOWN_START(I, uv_cooldown, I.uv_cooldown_length)
 
 /datum/wound/burn/treat(obj/item/I, mob/user)
-	if(istype(I, /obj/item/stack/medical/gauze))
-		bandage(I, user)
-	else if(istype(I, /obj/item/stack/medical/ointment))
+	if(istype(I, /obj/item/stack/medical/ointment))
 		ointment(I, user)
 	else if(istype(I, /obj/item/stack/medical/mesh))
 		mesh(I, user)
 	else if(istype(I, /obj/item/flashlight/pen/paramedic))
 		uv(I, user)
 
-/// basic support for instabitaluri/synthflesh healing flesh damage, more chem support in the future
-/datum/wound/burn/proc/regenerate_flesh(amount)
+// people complained about burns not healing on stasis beds, so in addition to checking if it's cured, they also get the special ability to very slowly heal on stasis beds if they have the healing effects stored
```

Comment:
????

Stasis bed is for stasis

Reply:
discussed this with you on dsicord, it's just a little bit of cheating so people don't keep filing bugs about wounds not healing until people are unbuckled